Makefile.pre.in

```diff
@@ -936,12 +936,9 @@ $(BUILDPYTHON)-gdb.py: $(SRC_GDB_HOOKS)
 # known modifications to the source tree before building. The patch will be
 # applied in a dry-run mode (validating, but not applying the patch) on builds
 # that *have* a compliance patch, but where compliance has not been enabled.
```

Comment:
You'll have to amend this comment to align it with the new behaviour :)

I may also be worth mentioning that this target is added to `INSTALLTARGETS` by the configure script.